pkg/actions/karpenter/karpenter.go

```diff
@@ -0,0 +1,53 @@
+package karpenter
+
+import (
+	"time"
+
+	"github.com/weaveworks/eksctl/pkg/karpenter"
+	"github.com/weaveworks/eksctl/pkg/utils/waiters"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"k8s.io/client-go/kubernetes"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/eks"
+)
+
+type Manager struct {
```

Comment:
```suggestion
type Installer struct {
```

i'm guilty of using `manager` everywhere, but here if its only installing I think it makes more sense?

Reply:
But the struct is called `Installer` :D

Reply:
> But the struct is called `Installer` :D

:thinking: I don't follow?

Reply:
Huh, I meant to rename the interface. :D :/

Reply:
Wait... I did. What is going on here? It shows to me that you Renamed the `Installer struct` from Manager. Which is super weird because the interface is called manager and the struct is called Installer.

Reply:
I was looking at the wrong file. :D